nodejs/Dockerfile

```diff
@@ -1,14 +1,13 @@
 FROM ubuntu
 MAINTAINER Kimbro Staken
 
-RUN apt-get install -y python-software-properties python
+RUN apt-get update
```

Comment:
According to [Docker best-practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/), you should not run `apt-get update` in a separate `RUN` step before `apt-get install`.

The reason is that each `RUN` step is creating a new docker container.
Therefore, when you `RUN apt-get install ...`, the new container does not know about the previous `RUN apt-get update`.

Instead, you should chain them together, like so:
```
RUN apt-get update && apt-get install -y ...
```

Lastly, to keep container sizes at a minimum, it's always a good idea to delete the cached apt-get source lists/packages with `rm -rf /var/lib/apt/lists/*`

So, your `RUN` step should look something like:
```
...
RUN add-apt-repository ppa:chris-lea/node.js && \
         echo "deb http://us.archive.ubuntu.com/ubuntu/ precise universe" >> /etc/apt/sources.list && \
         apt-get update && \
         apt-get install -y \
         software-properties-common \
         python \
         nodejs
RUN mkdir /var/www
...
```